packages/roosterjs-content-model-dom/lib/modelApi/creators/createListItem.ts

```diff
@@ -15,14 +15,18 @@ export function createListItem(
     levels: ContentModelListLevel[],
     format?: ContentModelSegmentFormat
 ): ContentModelListItem {
+    const formatHolder = createSelectionMarker(format);
+
+    formatHolder.isSelected = false;
```

Comment:
if the marker is in the list item, should it be true?

Reply:
Actually we never read that value, so it can be false.